.github/workflows/build-crowdin.yml

```
@@ -0,0 +1,30 @@
+name: Build Crowdin project
+
+on:
+  schedule:
+    - cron: "20 04 1 * *"
```

Add comment to make this human readable
```suggestion
    - cron: "20 04 1 * *" # Runs at 4:20 AM on the first day of every month
```